docs/examples/bosh-deployment/boshdeployment-with-custom-variable-disable-sidecar.yaml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
If you write an example file, could you add an e2e case for it?
https://ci.flintstone.cf.cloud.ibm.com/builds/92928#L5d27a265:531

Reply:
yes, i missed it, working on it